app/models/setting/auth.rb

```diff
@@ -27,7 +27,7 @@ def self.load_defaults
         self.set('websockets_ssl_key', N_("Private key that Foreman will use to encrypt websockets "), nil),
         self.set('websockets_ssl_cert', N_("Certificate that Foreman will use to encrypt websockets "), nil),
         # websockets_encrypt depends on key/cert when true, so initialize it last
-        self.set('websockets_encrypt', N_("VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required)"), SETTINGS[:require_ssl]),
+        self.set('websockets_encrypt', N_("VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required)"), !!SETTINGS[:require_ssl]),
```

Comment:
Yeah, or set it to some non-boolean value - since we guess the settings_type from the class of the default value, I thought it better to be sure it's a boolean in both here and the DB migration.